client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -173,6 +179,24 @@ class SharingServiceDescription extends Component {
 				args: { service: this.props.service.label },
 				context: 'Sharing: Publicize',
 			} );
+		} else if ( 'refresh-failed' === this.props.status ) {
+			if ( this.props.expires ) {
+				description = this.props.translate(
+					'We are unable to refresh your %(service)s token. Please reconnect before %(expiryDate)s',
+					{
+						args: {
+							service: this.props.service.label,
+							expiryDate: this.props.moment( this.props.expires * 1000 ).format( 'll' ),
+						},
+						context: 'Sharing: Publicize',
+					}
+				);
+			} else {
+				description = this.props.translate( 'We are unable to refresh your %(service)s token.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?